horovod/common/parameter_manager.cc

```diff
@@ -213,49 +192,41 @@ void ParameterManager::Tune(double score) {
     }
 
     // Send the updated parameter values to other workers.
-    SyncParams();
+    return true;
```

Comment:
This is slightly different behavior.  In the old code, we would call `Reset()` regardless of whether we `SyncParams()` or not, but here we only reset when we don't sync.  We should be sure to `Reset()` as part of synchronizing the parameters.

Reply:
Yes. The `Reset()` is called inside sync function at `mpi_controller.cc:211`.